### PR TITLE
send one packet at a time for the lockdown tunnel

### PR DESCRIPTION
**Pull request by dmissmann**
the TUN interface often returned an error when an incomplete IPv6 packet was sent (one IPv6 packet + parts of the next one) sending one packet at a time resolved this issue

### CODE REVIEW

File: ios/tunnel/tunnel_lockdown.go [hunk 1]
```diff
@@ -87,21 +89,38 @@ func forwardTUNToDevice(ctx context.Context, mtu uint64, tun io.Reader, deviceCo
...
 		case <-ctx.Done():
 			return nil
 		default:
-			n, err := deviceConn.Read(b)
+			_, err := io.ReadFull(br, ip6Header)
+			if err != nil {
+				return fmt.Errorf("failed to read IPv6 header: %w", err)
+			}
+
+			if ip6Header[0] != 0x60 {
```
Comment (danielpaulus) [inline ios/tunnel/tunnel_lockdown.go:109]: @dmissmann  I think IPv4 can happen here, no big deal and should not need an error

Comment (dmissmann) [reply ios/tunnel/tunnel_lockdown.go:109]: Currently that shouldn't be possible, we configure the TUN interface only with an IPv6 https://github.com/danielpaulus/go-ios/blob/main/ios/tunnel/tunnel.go#L184

Comment (danielpaulus) [reply ios/tunnel/tunnel_lockdown.go:109]: hmm ok, but I definitely got some ipv4 packets on mac os when testing this.

Comment (danielpaulus) [reply ios/tunnel/tunnel_lockdown.go:109]: I might add code to just drop ipv4 then later.

**Review by danielpaulus — APPROVED** [review]
nice!

**Pull request merged**